portfolio/src/main/webapp/work.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Work Experience</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>Work Experience</h1>
+      
+        <div class="work">Cluster Technician Coordinator – Student Technology Collaborative
+            <br>
+            February 2019 - present
+            <br>
+        Yale University, New Haven, CT</div>
+        <div class="description">Overseeing all shifts of Cluster Technicians on campus, contacting appropriate resources to resolve computer
+        errors, issuing warnings to Cluster Technicians not fulfilling job requirements, and collaborating daily with
+        other Coordinators to ensure that computer clusters are adequately maintained.<br><br></div>
+        <div class="work">Teaching Assistant – CS50
+            <br>
+            May 2019 - June 2019
+            <br>
+        Yale University, New Haven, CT</div>
+        <div class="description">Delivered key insights to over thirty students taking CS50 - Introduction to Computer Science - during open
+        office hours held twice a week. Held weekly meetings and lunches related to coursework, and graded
+        problem sets in C, Python, HTML, CSS, and JavaScript.<br><br></div>
+        <div class="work">Cluster Technician – Student Technology Collaborative
+            <br>
+            October 2018 - February 2019
+            <br>
+        Yale University, New Haven, CT</div>
+        <div class="description">Supported and maintained cluster computers on campus, including ones in residential colleges, public
+        clusters, and other departmental clusters to ensure that everyone on Yale’s campus always had access to
+        printing.<br><br></div>
+        <div class="work">Math Instructor – Mathnasium
+            <br>
+            August 2015 – August 2018
+            <br>
+        Rancho Santa Margarita, CA</div>
+        <div class="description">Tutored and mentored students in kindergarten through the 12th grade to learn, catch up, and get ahead in
+        math. Taught all math subjects up to and including AP Calculus and AP Statistics.<br><br></div>
+
+        <button type="submit" onclick="window.open('/resources/resume.pdf')">Download my resume</button>
```

Comment:
JS, woohoo! As an FYI:

This is a small JS function so it's totally cool to keep it inline in the HTML. In the future, as things get more complex, consider defining JS functions in the script file and then calling those functions in the HTML.

Reply:
Sure thing. I'll separate this function as a small practice in the next pull request.

Reply:
+1 to putting JS in a separate file, however in this case you might also be able to replace the button with the onclick with just a `<a href=...` tag.  That has some advantages, for example it will show the link in bottom left status bar, and lets the browser add right click actions like "Open Link in a New Tab" or "Copy Link Address", which don't work with the onclick style

Reply:
Ah, I see. I'll look into this.